web/files/files.go

```diff
@@ -58,6 +58,21 @@ const TagSeparator = ","
 // recognized
 var ErrDocTypeInvalid = errors.New("Invalid document type")
 
+// DrivesCreationHandler is the handler for POST /files/drives. It creates the
+// directory where shared and external drives are saved if it doesn't exist,
+// and return information about this directory.
+func DrivesCreationHandler(c echo.Context) error {
```

Comment:
I think the route and its handler should contain the word `shared` as well to be better convey the intent.